client/src/rpc_client.rs

```diff
@@ -320,7 +320,7 @@ impl RpcClient {
         loop {
             // Retry for about 25s because default behavior uses `CommitmentLevel::Max`
             println!("retry sending transaction");
-            let mut status_retries = 50;
+            let mut status_retries = 100;
```

Comment:
It would be interesting to see what root slot the bootstrap node is at when the airdrop tx is submitted and throughout the retries.